src/panels/ConfigPanel/types.ts

```diff
@@ -44,3 +46,22 @@ export enum Label {
 export enum TestId {
   PANEL = "config-panel",
 }
+
+export enum InlineErrors {
+  FORM = "form",
+  GET_CONFIG = "get-config",
+  SUBMIT_TO_JUJU = "submit-to-juju",
```

Comment:
`FORM` and `SUBMIT_TO_JUJU` are only used inside ConfirmationDialog so they should be moved to a types file for that component.

It looks like a lot of of the values in the `Label` can also be moved to the new component.

Reply:
Nice catch! Everything that was no longer used in `ConfigPanel` was moved to `ConfirmationDialog`. Thanks for the review.